src/reports.c

```diff
@@ -1433,9 +1433,16 @@ static void cb_add_seen(region *r, unit *u, void *cbdata) {
 void report_warnings(faction *f, int now)
 {
     if (f->age < NewbieImmunity()) {
```

Comment:
NewbieImmunity könnte man auch als `int NewbieImmunity(faction *)` definieren. Je weniger auf f->age zugegriffen wird, desto besser. Ich würde das gern irgendwann mal mit einem f->created ersetzen, damit nicht jede Woche f->age++ gemacht werden muss...

Reply:
Es gibt doch schon IsImmune(faction *) in laws.c. Sollte man dann hier einfach übernehmen.

Reply:
Ist nicht so wichtig, kann ich immer nochmal machen, wenn ich f->age wirklich rauswerfe.